images/dev/Dockerfile.dev

```
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9-minimal:9.7
+FROM registry.access.redhat.com/ubi9-minimal:9.8
 
 ADD artifacts/configuration.yaml /etc/clusterresourceoverride/config/override.yaml
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)